filters/ThirdParty/filter_107_EasyListGermany/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 107,
   "name": "EasyList Germany",
-  "description": "German supplement for EasyList. Already included in AdGuard German filter.",
+  "description": "Additional filter list for websites in Germany. Already included in AdGuard German filter.",
```

Comment:
Additional filter list for websites in German. Already included in AdGuard German filter.